libs/blocks/section-metadata/section-metadata.js

```diff
@@ -27,6 +27,17 @@ export async function handleStyle(text, section) {
   section.classList.add(...styles);
 }
 
+function handleMasonry(text, section) {
+  section.classList.add(...['masonry-layout', 'course-up']);
```

Comment:
What is `course-up`?

Reply:
@rgclayton The class is not adding any new CSS. However it allows us to use the customizable gap customization option already available in the section metadata for masonry as well - `.section[class*='-up'].<size>-gap`

Reply:
@rgclayton I have update the classname to make it more intuitive.